CHANGES.rst

```diff
@@ -29,6 +30,7 @@
 - Raise ``NoSuchTableError`` when trying to reflect a table that doesn't exist.
   (`Issue #38 <https://github.com/graingert/redshift_sqlalchemy/issues/38>`_)
 
+
```

Comment:
The newline as an attempt to make the whitespacing consistent. Other sections in this file are preceeded by two blank lines. It looks like I missed adding an extra blank line beneath the 0.2.1 section while I was at it.

It doesn't matter functionally, so I'll revert the change to avoid noise in the diff.